help/mastering-the-compose-box.md

```diff
@@ -8,7 +10,38 @@ look up a related discussion.
 In this context, the parts of the message view that are outside of the
 conversation you are composing to are faded for clarity.
 
-## Go to conversation
+### Change where you are composing to
+
+No matter where you start composing your message, it's easy to change where it
+will be sent.
+
+{start_tabs}
+
+{tab|stream-compose}
+
+{!start-composing.md!}
+
+1. Select any stream from the dropdown in the top left of the compose box. You can
+   start typing to filter streams.
+
+1. Enter a topic name. Auto-complete will provide suggestions for previously
+   used topics.
+
+{tab|dm-compose}
+
+{!start-composing.md!}
+
+1. Select **Direct message** from the dropdown in the top left of the compose
+   box.
+
+1. Start typing the name of the person you want to message, and
+   select their name from the list of suggestions. You can continue
+   adding as many message recipients as you like.
+
+{end_tabs}
+
+
+### Go to conversation
 
 Zulip lets you narrow the message view to the conversation you're currently
 composing to.
```

Comment:
I like this being placed just after the previous section, since this is often something you might want to do after doing the previous. It's possible we should add more of a textual transition here.